packages/core/strapi/lib/services/entity-validator/validators.js

```diff
@@ -213,4 +214,5 @@ module.exports = {
   time: composeValidators(addUniqueValidator),
   datetime: composeValidators(addUniqueValidator),
   timestamp: composeValidators(addUniqueValidator),
+  blocks: () => StrapiBlocksSchema,
```

Comment:
Not sure how this object works but it's odd that this entry doesn't match any other